components/ClassCard/ClassCard.tsx

```diff
@@ -44,9 +44,12 @@ const ClassCard = ({
 
     const classColorRGB = (dark: boolean) => `rgb(${hexWithOpacityToRgb(_class.color, 0.6, dark ? 0 : 255).join(",")})`;
 
+    const isInThePast = new Date(_class.from).getTime() < new Date().getTime();
```

Comment:
Could have used `useMemo` here, simply because the value rarely changes, but probably no noticeable difference.